### ISSUE TITLE
Stackage nightly build failure

### ISSUE THREAD

**Issue by decentral1se** [kkazuo/xlsx-tabular#8]
Saw for xlsx-tabular-0.2.2:

```
src/Codec/Xlsx/Util/Tabular/Json.hs:22:1: error:
    <E2><80><A2> No instance for (FromJSON ErrorType)
        arising from a use of <E2><80><98>parseJSON<E2><80><99>
    <E2><80><A2> In the second argument of <E2><80><98>(<$>)<E2><80><99>, namely <E2><80><98>(parseJSON arg_arzm)<E2><80><99>
      In the expression: (CellError <$> (parseJSON arg_arzm))
      In a case alternative:
          arg_arzm -> (CellError <$> (parseJSON arg_arzm))
   |
22 | deriveJSON defaultOptions
   | ^^^^^^^^^^^^^^^^^^^^^^^^^...

src/Codec/Xlsx/Util/Tabular/Json.hs:22:1: error:
    <E2><80><A2> No instance for (ToJSON ErrorType) arising from a use of <E2><80><98>toJSON<E2><80><99>
    <E2><80><A2> In the second argument of <E2><80><98>(.=)<E2><80><99>, namely <E2><80><98>(toJSON arg1_arz3)<E2><80><99>
      In the expression:
        ((Data.Text.pack "contents") .= (toJSON arg1_arz3))
      In the second argument of <E2><80><98>(:)<E2><80><99>, namely
        <E2><80><98>[((Data.Text.pack "contents") .= (toJSON arg1_arz3))]<E2><80><99>
   |
22 | deriveJSON defaultOptions
   | ^^^^^^^^^^^^^^^^^^^^^^^^^...

src/Codec/Xlsx/Util/Tabular/Json.hs:22:1: error:
    <E2><80><A2> No instance for (ToJSON ErrorType)
        arising from a use of <E2><80><98>aeson-1.2.3.0:Data.Aeson.Types.ToJSON.toEncoding<E2><80><99>
    <E2><80><A2> In the second argument of <E2><80><98>(Data.Aeson.Encoding.Internal.><)<E2><80><99>, namely
        <E2><80><98>(aeson-1.2.3.0:Data.Aeson.Types.ToJSON.toEncoding arg1_arz9)<E2><80><99>
      In the second argument of <E2><80><98>(Data.Aeson.Encoding.Internal.><)<E2><80><99>, namely
        <E2><80><98>(Data.Aeson.Encoding.Internal.colon
            Data.Aeson.Encoding.Internal.><
              (aeson-1.2.3.0:Data.Aeson.Types.ToJSON.toEncoding arg1_arz9))<E2><80><99>
      In the expression:
        ((Data.Aeson.Encoding.Internal.string "contents")
           Data.Aeson.Encoding.Internal.><
             (Data.Aeson.Encoding.Internal.colon
                Data.Aeson.Encoding.Internal.><
                  (aeson-1.2.3.0:Data.Aeson.Types.ToJSON.toEncoding arg1_arz9)))
   |
22 | deriveJSON defaultOptions
   | ^^^^^^^^^^^^^^^^^^^^^^^^^...
```

**Comment by kkazuo** [comment]
Fixed on version 0.2.2.1 https://hackage.haskell.org/package/xlsx-tabular-0.2.2.1

**Comment by decentral1se** [comment]
Thanks!
